Intersect.Editor/Forms/Editors/Events/Event Commands/EventCommand_Warp.cs

```diff
@@ -67,9 +67,9 @@ private void InitLocalization()
             lblDir.Text = Strings.Warping.direction.ToString("");
             btnVisual.Text = Strings.Warping.visual;
             cmbDirection.Items.Clear();
-            for (var i = -1; i < 4; i++)
+            for (var i = Direction.None; i < (Direction)4; i++)
```

Comment:
Direction i = 0, and If the cast from 4 isn't needed, remove

Reply:
https://github.com/AscensionGameDev/Intersect-Engine/pull/1708#discussion_r1125942002